operators/derive/to_es.py

```diff
@@ -66,6 +70,54 @@ def data_api_es_flow():
                 },
             },
         ),
+        DF.set_type(
```

Comment:
I think we're probably better off marking all of these as `es:index: False` - I don't see us querying based on any of these fields. wdyt @pwalsh ?

Reply:
@akariv sure, I'll update,

Reply:
in https://github.com/whiletrue-industries/srm-etl/commit/704d11464b93056d0d12cf1d5a9b147d9b278ba1